e2e/cypress/e2e/languages/languageCreationNewProject.cy.ts

```diff
@@ -110,9 +110,12 @@ describe('Language creation in new project', () => {
 
   it('validates language repeat', () => {
     getInputByName('name').type('Super project');
-    addLanguage('English');
-    gcy('global-form-save-button').click();
-    cy.contains('Cannot add language more than once').should('be.visible');
+    typeToAutocomplete('English');
+    gcy('languages-create-autocomplete-suggested-option')
+      .contains('English')
+      .should('have.css', 'pointer-events', 'none')
+      .click({ force: true });
```

Comment:
Click fails if there is 'pointer-events: none'. However I want to test, that it's really disabled, so I force click it and then check that nothing happened.

Reply:
Hmm, but it doesn't test it much anyway, since there is no waiting. Even if it was clickable, there is a chance cypress is faster. Not sure if there is sane way to test it without waiting. I would be fine just with the css property check.